### PR TITLE
[CI/CD] Setup continuous integration

### PR DESCRIPTION
**Pull request by abuccts**
Setup continuous integration, add travis ci configuration.

### CODE REVIEW

**Comment by abuccts** [comment]
pls view ci results in [my forked version](https://travis-ci.org/abuccts/pai/builds/340836305)

**Comment by fanyangCS** [comment]
@ydy is added to the review. #Closed

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+sudo: required
+dist: trusty
+matrix:
```
Comment (fanyangCS) [inline .travis.yml]: >matrix [](start = 0, length = 6)

@ydy, can you include python in?

Comment (ydye) [reply .travis.yml]: https://github.com/codecov/example-python/blob/master/.travis.yml
https://docs.travis-ci.com/user/languages/python/#Default-Build-Script

**Pull request merged**